File: challenge102/challenge102.py
"""
To use the python script in Command Prompt:
Write 'python your_file_name.py'
"""
# Function that takes a string
def NameShuffle(name):
    # Splits the string parameter into indexes in an arrray
    fullname = name.split(' ')
    # assigns 'first' to the value of index [0]
    first = fullname[0]
    # assigns 'last' to the value of index[1]
    last = fullname[1]
    # prints the variables 'last' and 'first'
    # with the value from the index numbers [1] and [0]
    print(f"{last} {first}")

# function calls with 1 string as argument
NameShuffle("Michael Hansen")
NameShuffle("Joe Biden")
NameShuffle("Samwise Gamgee")
NameShuffle("Obi-wan Kenobi")

File: challenge102/challenge102.js
/*
To use the script in Command Prompt:
write 'node your_file_name.js'
*/

// Function that takes a string
function NameShuffle(name){
    // splits the string, and assigns the array to 'fullname'
    var fullname = name.split(' ');
    // logs 'fullname' with the values of indexes [0] and [1]
    console.log(fullname[1] + ' ' + fullname[0]);
}

// Function calls with arrguments
NameShuffle("Michael Hansen");
NameShuffle("Barack Obama");
NameShuffle("Frodo Baggins");
NameShuffle("Anakin Skywalker");